src/sota_tools/deploy.cc

```diff
@@ -67,7 +67,7 @@ void uploaded_ev(RequestPool &p, OSTreeObject::ptr h) {
 }
 
 bool copy_repo(const std::string &cacerts, const std::string &src, const std::string &dst, const std::string &ref,
-               const std::string &hardwareids, bool sign, bool dryrun) {
+               const std::string &hardwareids, bool sign, bool is_offline_signed, bool dryrun) {
```

Comment:
I don't think `is_offline_signed` is necessary: Whether to use offline signing can be determined by ServerCredentials::CanSignOffline(), and there isn't (to my knowledge) a reason for the user to override that choice.